core/src/main/scala/package.scala

```diff
@@ -226,13 +226,13 @@ package object quiver {
 
   /** Find the first path in a search tree that starts with the given node */
   def getPath[N](v: N, t: RTree[N]): Option[Path[N]] =
-    t.find(_.head == v).map(_.reverse)
+    t.find(_.headOption == Some(v)).map(_.reverse)
```

Comment:
Small pet hate: can we possibly make `Some` -> `Option`. There are a few cases around `null` handling for which `Some.apply` is not safe.